src/main/resources/assets/quark/lang/ru_ru.json

```
@@ -1512,62 +1504,62 @@
     "advancements.quark.get_forgotten_hat.title": "Это принадлежит музею!",
-    "advancements.quark.get_forgotten_hat.description": "Убейте Забытого под землёй и заберите его шляпу.",
+    "advancements.quark.get_forgotten_hat.description": "Убейте забытого под землёй и заберите его шляпу",
     "advancements.quark.get_diamond_heart.title": "Сердечный приступ",
-    "advancements.quark.get_diamond_heart.description": "Подкрадитесь к стоунлингу под землёй и убейте его, чтобы получить его сердце.",
-    "advancements.quark.make_stoneling.title": "Молодой мечтатель",
-    "advancements.quark.make_stoneling.description": "Создайте своего собственного стоунлинга, используя алмазное сердце на любом камне.",
+    "advancements.quark.get_diamond_heart.description": "Подкрадитесь к стоунлингу под землёй и убейте его, чтобы получить его сердце",
+    "advancements.quark.make_stoneling.title": "Жизнь камня",
+    "advancements.quark.make_stoneling.description": "Создайте своего собственного стоунлинга, используя алмазное сердце на любом камне",
     "advancements.quark.mine_toretoise.title": "Моя черепаха!",
     "advancements.quark.mine_toretoise.description": "Добудьте руду с пещерепахи киркой",
     "advancements.quark.mine_fed_toretoise.title": "Мы богаты!",
-    "advancements.quark.mine_fed_toretoise.description": "Добудьте руду с пещерепахи после того, как она восстановит свою руду от поедания светящихся ягод.",
-    "advancements.quark.throw_pickarang.title": "Каким-то образом подбиранг вернулся",
-    "advancements.quark.throw_pickarang.description": "Бросить подбиранг",
-    "advancements.quark.retreive_flamerang.title": "Лети в небо, огненная птица",
-    "advancements.quark.retreive_flamerang.description": "Бросьте пламеранг и верните его, пока он несет горящий предмет.",
-    "advancements.quark.overlevel_enchant.title": "Нарушитель лимита",
-    "advancements.quark.overlevel_enchant.description": "Превысьте уровень чар с помощью древних томов",
-    "advancements.quark.instamine_deepslate.title": "Мгновенное нарушение лимита",
-    "advancements.quark.instamine_deepslate.description": "Мгновенно добудьте глубинный сланец с помощью Эффективности VI и Спешки II",
+    "advancements.quark.mine_fed_toretoise.description": "Добудьте руду пещерепахи после того, как она отрастит свою руду от поедания светящихся ягод",
+    "advancements.quark.throw_pickarang.title": "Каким-то образом возвращается",
+    "advancements.quark.throw_pickarang.description": "Бросьте подбиранг",
+    "advancements.quark.retreive_flamerang.title": "Лети, огненная птица",
+    "advancements.quark.retreive_flamerang.description": "Бросьте пламеранг и поймайте его, пока он несет горящий предмет",
+    "advancements.quark.overlevel_enchant.title": "Запретная магия",
+    "advancements.quark.overlevel_enchant.description": "Поднимите уровень зачарования выше максимального его значения при помощи древнего фолианта",
```

```suggestion
    "advancements.quark.overlevel_enchant.description": "Поднимите уровень чар выше их максимального значения при помощи древнего фолианта",
```
Комментарий о чарах есть на строке 248.